serverless.yml

```diff
@@ -45,6 +45,7 @@ functions:
     environment:
       STATE_MACHINE_ARN: arn:aws:states:${self:provider.region}:${self:custom.accountNumber}:stateMachine:aqts-capture-state-machine-${self:provider.stage}
       AWS_DEPLOYMENT_REGION: ${self:provider.region}
+      OBJECT_SIZE_LIMIT: 150000000
```

Comment:
This number might need some fiddling, but should be a good start.

Reply:
Nice that it's configurable here too.